src/services/ExposureNotificationService/ExposureNotificationService.ts

```diff
@@ -34,23 +34,29 @@ export {SystemStatus};
 export type ExposureStatus =
   | {
       type: 'monitoring';
-      lastCheckedPeriod?: number;
-      lastCheckedTimestamp?: number;
+      lastChecked?: {
```

Comment:
This is the missing piece. Although it does not affect functionality of the app but it is still good to have in term of type.